test/parse/test_element.rb

```diff
@@ -39,11 +39,11 @@ def test_empty_namespace_attribute_name
           parse("<x :a=\"\"></x>")
         end
         assert_equal(<<-DETAIL.chomp, exception.to_s)
-Invalid attribute name: <:a="">
+Invalid attribute name: <:a=""></x>>
```

Comment:
Can we use `:a` as the invalid attribute name?

Reply:
It cannot be used as an attribute name, which raises an `Invalid attribute name` exception.
As a test case, what is the problem?

Reply:
I wanted to say that can we use the following as the expected value?

```suggestion
Invalid attribute name: <:a>
```

Reply:
It is difficult to determine the appropriate character for an attribute given invalid attributes such as the following.

- `'<x :a=""></x>'`
- `'<x =""></x>'`
- `'<x ""></x>'`

Since it is possible to determine the tag end, we will continue to return `:a=""` as an invalid tag as before.